### PR TITLE
Add basic graphics for entities (Player)

### PR DESCRIPTION
**Pull request by dekrain**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -2,9 +2,14 @@ import SLW from './SLW'
...
 window.addEventListener('load', e => {
   let tileset = new Image
+  let entity_tileset = {}
 
   tileset.onload = () => {
-    const game = new SLW('1-1', tileset)
+    if (!(Object.keys(entity_tileset).every(img => entity_tileset[img].complete))) {
+      requestAnimationFrame(tileset.onload);
+      return;
+    }
```
Comment (dekrain) [inline src/index.js:11]: Make sure all images are loaded

File: src/index.js [hunk 2]
```diff
@@ -30,5 +35,7 @@ window.addEventListener('load', e => {
     })()
   }
 
+  entity_tileset.Player new Image
```
Comment (towerofnix) [inline src/index.js]: I'd imagine this is a syntax error?

Comment (dekrain) [reply src/index.js]: Oops

**Comment by bates64** [comment]
Merged this manually a few commits ago.

**Comment by dekrain** [comment]
On 12907eaa38a56056854f0935d739da7b0ca9405d, I know

**Pull request closed without merging**